webui/react/src/App.module.scss

```diff
@@ -41,10 +42,14 @@
       [class^=NavigationTabbar_base] { display: flex; }
       & > main {
         height: calc(100vh - var(--theme-sizes-navigation-topbarHeight) - var(--theme-sizes-navigation-toolbarHeight));
```

Comment:
keeping this as a fallback or should it be removed?

Reply:
Ah good point, I guess we don't need the fallback because we already rely on css variables elsewhere for theme variables. Also everything supports it now other than the old IE, which we explicitly decided not to support.